cmake fails on systems without lsb_release

```
CMake Error at CMakeLists.txt:21 (if):
  if given arguments:

    "STREQUAL" "vivid"

  Unknown arguments specified
```
Same as https://github.com/ubports/unity-scopes-api/issues/4

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,7 +18,7 @@ if (NOT DEFINED TRUST_STORE_VERSION_MAJOR)
   #   - other distros
   #   - errors
   # we define the version to be 2.0.0
-  if (${DISTRO_CODENAME} STREQUAL "vivid")
+  if (DISTRO_CODENAME STREQUAL "vivid")
     set(TRUST_STORE_VERSION_MAJOR 1)
     set(TRUST_STORE_VERSION_MINOR 1)
     set(TRUST_STORE_VERSION_PATCH 0)

